### ISSUE TITLE
Line break issues

### ISSUE THREAD

**Issue by HarrisonHenri** [smalot/pdfparser#294]
First of all, thank you very much for the excellent library! So, I have a PDF in this format:

![image](https://user-images.githubusercontent.com/13942973/83132593-ff3f0180-a0b7-11ea-97ee-4b9a684bc7bc.png)

And when parsing it for text, the line break is lost. Any suggestion?

Thanks in advance!

**Comment by k00ni** [comment]
Hi @HarrisonHenri, you mean that all the text is on the same line? Can you provide us an example PDF so we can reproduce it locally?

**Comment by HarrisonHenri** [comment]
I mean that when the string is returned by the parser, I can't break it using the \n delimiter. For example, I got that for the file that I sent above:

![image](https://user-images.githubusercontent.com/13942973/83136685-46c88c00-a0be-11ea-9c84-caaefcb2ed16.png)

Example PDF

[0a330d756122035698aeb23e33d10621.pdf](https://github.com/smalot/pdfparser/files/4695200/0a330d756122035698aeb23e33d10621.pdf)

**Comment by rubenvanerk** [comment]
I can confirm I have the same issue. Every gets just one line instead of multiple lines.

What is really interesting is that when I changed the number on this line:
https://github.com/smalot/pdfparser/blob/123d56c25d4e9a2b9f37e383718da515e2436f1c/src/Smalot/PdfParser/PDFObject.php#L346
To something lower then 10, the file parsed perfectly. For this document the value of `$delta` appears to be 9.84.

**Comment by j0k3r** [comment]
Should we _round_ `$delta` to avoid it?

**Comment by rubenvanerk** [comment]
That would solve this case, yes. I don't know what the impact for other cases would be though.

**Comment by k00ni** [comment]
Correct me when i am wrong: A certain threshold in `$delta` decides if line breaks are lost or not?

**Comment by rubenvanerk** [comment]
Yes, lowering the value against which `$delta` is compared to _anything_ below 9.84 makes it so that line breaks are not lost in this case.

**Comment by k00ni** [comment]
What are our options here to solve/avoid the issue in the future?

**Comment by rubenvanerk** [comment]
- Round `$delta` as @j0k3r suggested
- Lower the value against which `$delta` is compared

Options above _could_ affect how well other pdfs are parsed so another option could be: 
- Allow the user to overwrite the value against which `$delta` is compared

**Comment by pedrofeliz90** [comment]
I changed the delta here to> 9.84,> 9, and none worked, they all leave the text in just 1 line.
